src/ziggurat/server/middleware.clj

```diff
@@ -31,3 +33,12 @@
       (catch Exception ex
         (sentry/report-error sentry-reporter ex "Uncaught error in server")
         {:status 500 :body (json/encode {:Error (st/pst-str ex)})}))))
+
+(defn publish-metrics [handler]
+  (fn [request]
+    (let [response (handler request)
+          request-uri       (:uri request)
+          response-status   (:status response)
+          service-name      (:app-name (ziggurat-config))]
+      (metrics/increment-count [service-name "http" "stats"] "count" {:request-uri request-uri :response-status response-status})
```

Comment:
if we are adding service_name in tags, there's no need to add it to the metric namespace

Reply:
we can have a better name here - 'http-server' 'responses' , maybe